web/projects/em/src/app/settings/security/security-provider/authentication-provider-detail-view/authentication-provider-detail-view.component.ts

```diff
@@ -46,12 +46,14 @@ export class AuthenticationProviderDetailViewComponent implements OnInit, OnDest
 
    @Input()
    set model(model: AuthenticationProviderModel) {
-      this._model = model;
-      this._original = Tool.clone(model);
+      if(model == null || !!model.providerType) {
```

Comment:
How does this make sense? It's only setting the model if the parameter is null or the provider type is not set.